packages/sdk/src/__tests__/engine-commands/getConfig.test.ts

```diff
@@ -105,4 +105,105 @@ describe('getConfig', () => {
 
     expect(config).toMatchSnapshot()
   })
+  test('with engineType', async () => {
```

Comment:
Maybe split in 2 tests? one engineType=binary and the other engineType=library?

Reply:
Done